### ISSUE TITLE
Reduce requirements list in setup.cfg

### ISSUE THREAD

**Issue by kwinkunks** [jupyter/nbclassic#252]
Thanks for continuing to support classic notebooks, and improving the architecture :raised_hands: 

Currently the `install_requires` list is comprehensive:

https://github.com/jupyter/nbclassic/blob/v1.0.0/setup.cfg#L30-L49

But most of these requirements are requirements of `jupyter-server`, and that is a requirement of `notebook-shim` so the list could be substantially reduced to:

```
install_requires:
    ipykernel
    ipython-genutils
    nest-asyncio
    notebook-shim
```

Personally I think this makes it easier for people to figure out their dependency requirements, but I realize the team may have another viewpoint.

**Comment by echarles** [comment]
I agree with you @kwinkunks By any chance, would you have appetite to open a PR for this. I would be very happy to review and help you with that.
